CRA preset broken on Windows

The preset works fine on macOS, however on Windows met with an error when trying to `start-storybook`:

```
ERR! Error: Cannot find module 'C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\config\webpack.config'
ERR! Require stack:
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\preset-create-react-app\index.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\presets.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\config.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\build-static.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\server.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\react\dist\server\index.js
ERR! - C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\react\bin\index.js
ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:780:15)
ERR!     at Function.Module._load (internal/modules/cjs/loader.js:685:27)
ERR!     at Module.require (internal/modules/cjs/loader.js:838:19)
ERR!     at require (internal/modules/cjs/helpers.js:74:18)
ERR!     at Object.webpack (C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\preset-create-react-app\index.js:55:28)
ERR!     at C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\presets.js:82:72
ERR!     at async Promise.all (index 1)
ERR!     at async buildDevStandalone (C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\build-dev.js:332:9)
ERR!     at async buildDev (C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\@storybook\core\dist\server\build-dev.js:411:3)
ERR!  Error: Cannot find module 'C:\Projects\Anglo American\AD Core\da-ad-core-ui\node_modules\config\webpack.config'
...
```

It appears as though `getReactScriptsPath()` isn't returning what is expected on Windows.

Tested on both Node v10.16.3 and v12.11.1.

Using `react-scripts@3.1.2`.

--- a/packages/preset-create-react-app/README.md
+++ b/packages/preset-create-react-app/README.md
@@ -1,4 +1,4 @@
-# TypeScript preset for Storybook
+# Create React App preset for Storybook
 
 One-line [Create React App](https://github.com/facebook/create-react-app) configuration for Storybook.
 

--- a/packages/preset-create-react-app/helpers/getReactScriptsPath.js
+++ b/packages/preset-create-react-app/helpers/getReactScriptsPath.js
@@ -4,16 +4,19 @@ const path = require('path');
 const getReactScriptsPath = () => {
   const cwd = process.cwd();
   const scriptsBinPath = path.join(cwd, '/node_modules/.bin/react-scripts');
+
   /*
    * Try to find the scripts package by following the `react-scripts` symlink.
    * This won't work for Windows users, unless within WSL.
    */
-  try {
-    const resolvedBinPath = fs.realpathSync(scriptsBinPath);
-    const scriptsPath = path.join(resolvedBinPath, '..', '..');
-    return scriptsPath;
-  } catch (e) {
-    // NOOP
+  if (process.platform !== 'win32') {
+    try {
+      const resolvedBinPath = fs.realpathSync(scriptsBinPath);
+      const scriptsPath = path.join(resolvedBinPath, '..', '..');
+      return scriptsPath;
+    } catch (e) {
+      // NOOP
+    }
   }
 
   /*

--- a/packages/preset-create-react-app/helpers/getReactScriptsPath.js
+++ b/packages/preset-create-react-app/helpers/getReactScriptsPath.js
@@ -5,25 +5,11 @@ const getReactScriptsPath = () => {
   const cwd = process.cwd();
   const scriptsBinPath = path.join(cwd, '/node_modules/.bin/react-scripts');
 
-  /*
-   * Try to find the scripts package by following the `react-scripts` symlink.
-   * This won't work for Windows users, unless within WSL.
-   */
-  if (process.platform !== 'win32') {
-    try {
-      const resolvedBinPath = fs.realpathSync(scriptsBinPath);
-      const scriptsPath = path.join(resolvedBinPath, '..', '..');
-      return scriptsPath;
-    } catch (e) {
-      // NOOP
-    }
-  }
-
-  /*
-   * Try to find the scripts package on Windows by following the `react-scripts` CMD file.
-   * https://github.com/storybookjs/storybook/issues/5801
-   */
   if (process.platform === 'win32') {
+    /*
+     * Try to find the scripts package on Windows by following the `react-scripts` CMD file.
+     * https://github.com/storybookjs/storybook/issues/5801
+     */
     try {
       const content = fs.readFileSync(scriptsBinPath, 'utf8');
       const packagePathMatch = content.match(
@@ -34,6 +20,18 @@ const getReactScriptsPath = () => {
         const scriptsPath = path.join(cwd, '/node_modules/.bin/', packagePathMatch[1]);
         return scriptsPath;
       }
+    } catch (e) {
+      // NOOP
+    }
+  } else {
+    /*
+     * Try to find the scripts package by following the `react-scripts` symlink.
+     * This won't work for Windows users, unless within WSL.
+     */
+    try {
+      const resolvedBinPath = fs.realpathSync(scriptsBinPath);
+      const scriptsPath = path.join(resolvedBinPath, '..', '..');
+      return scriptsPath;
     } catch (e) {
       // NOOP
     }

--- a/packages/preset-create-react-app/package.json
+++ b/packages/preset-create-react-app/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@storybook/preset-create-react-app",
-  "version": "1.0.0-beta.3",
+  "version": "1.0.0",
   "description": "Create React App preset for Storybook",
   "license": "MIT",
   "main": "index.js",

